tfhub_dev/assets/rishit-dagli/chest-xray-covid/default/tfjs/1.md

```diff
@@ -0,0 +1,18 @@
+# Tfjs rishit-dagli/chest-xray-covid/1/default/1
+TF.js deployment of [rishit-dagli/chest-xray-covid/1](https://tfhub.dev/rishit-dagli/chest-xray-covid/1/).
```

Comment:
Links are now allowed in the short model description (they are allowed in the full description starting below the model properties).

Reply:
Thank you for pointing that out, I have now fixed it.